kfs-core/src/main/resources/edu/arizona/kfs/sys/document/datadictionary/RouteCodeMaintenanceDocument.xml

```diff
@@ -0,0 +1,58 @@
+<beans xmlns="http://www.springframework.org/schema/beans"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:p="http://www.springframework.org/schema/p"
+    xmlns:dd="http://rice.kuali.org/dd"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans
+        http://www.springframework.org/schema/beans/spring-beans-2.0.xsd
+        http://rice.kuali.org/dd
+        http://rice.kuali.org/dd/dd.xsd">
+
+  <bean id="RouteCodeMaintenanceDocument" parent="RouteCodeMaintenanceDocument-parentBean" />
+
+  <bean id="RouteCodeMaintenanceDocument-parentBean" abstract="true" parent="MaintenanceDocumentEntry">
+    <property name="businessObjectClass" value="edu.arizona.kfs.sys.businessobject.RouteCode" />
+    <property name="maintainableClass" value="org.kuali.rice.kns.maintenance.KualiMaintainableImpl" />
```

Comment:
This still needs to be done:
Should be:
property name="maintainableClass" value="org.kuali.kfs.sys.document.FinancialSystemMaintainable"